source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -93,19 +105,29 @@ Set the client's initial TopologyType to ReplicaSetNoPrimary.
 (For most clients, parsing a connection string with a "replicaSet" option
 automatically sets the TopologyType to ReplicaSetNoPrimary.)
 
+Set up a listener to collect SDAM events published by the client, including
+events published during client construction.
+
 Test Phases
 ~~~~~~~~~~~
 
 For each phase in the file, parse the "responses" array.
 Pass in the responses in order to the driver code.
 If a response is the empty object `{}`, simulate a network error.
 
-Once all responses are processed, assert that the phase's "outcome" object
+For non-monitoring tests,
```

Comment:
Is the early line break intentional?

Reply:
Yes, it makes the file readable in a terminal as well as makes the diffs lower in size and more readable.